app/build.gradle.kts

```diff
@@ -71,6 +71,7 @@ dependencies {
     implementation(libs.navigation.compose)
     implementation(libs.androidx.security.crypto)
     implementation(libs.kotlinx.serialization.json)
+    implementation(libs.androidx.lifecycle.viewmodel.ktx)
```

Comment:
Why has a dependency been added? I cannot see it being used, nor a reason for why it should be included in the project.

Reply:
I got an unresolved reference `Unresolved reference: viewModelScope`. The only solution I found for this was adding the dependency to `build.gradle` : 

```
dependencies {
    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1"
}
```

Might have stumbled over wrong implementations, at least it fixed my unresolved issue x)